index-sample.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample</title>
+    <link rel="stylesheet" href="/style-sample.css">
```

Comment:
It's always better to use meaningful names in the code. May it be a title, filename, variable, classname, or a function name - It should have a meaningful name so that it'll be clear to a different developer going through the code.